examples/sampling/node_classification.py

```diff
@@ -289,7 +289,6 @@ def train(device, g, dataset, model, num_classes, use_uva, fused_sampling):
         args.mode = "cpu"
     print(f"Training in {args.mode} mode.")
 
-    # Load and preprocess dataset.
```

Comment:
Can you retype this comment so that the diff disappears?

Reply:
for sure